packages/bridge/src/vl/time-series/TimeSeries.ts

```diff
@@ -174,7 +166,23 @@ export class TimeSeries {
     });
   }
 
-  private _createAnimation() {
+  private _getAnimationExpression() {
+    if (this._variableName && this._viz.variables[this._variableName]) {
+      return this._viz.variables[this._variableName];
+    }
+
+    const expr =  this._propertyName && this._viz[this._propertyName].isAnimated()
+      ? this._viz[this._propertyName]
+      : this._createDefaultAnimation();
```

Comment:
Could we use in the `else` the following expression? `viz[prop] AND defaultAnimation`?

Reply:
I don't understand this comment, it might be outdated by now :eyes: 

You can't use viz[this._propertyName] in the else clause cause it might be null-ish, right?

Reply:
you're right, it can be if the property name is not a style property 👌